tests/nightly/model_backwards_compatibility_check/JenkinsfileForMBCC

```diff
@@ -55,7 +55,7 @@ core_logic: {
     node(NODE_LINUX_CPU) {
       ws('workspace/modelBackwardsCompat') {
         utils.init_git()
```

Comment:
Please remove this line (init_git)

Reply:
Done in : dd8856e460381152b021f62240c9b6cc99729078